easybuild/easyconfigs/m/M4/M4-1.4.18.eb

```diff
@@ -16,12 +16,18 @@ toolchain = {'name': 'dummy', 'version': 'dummy'}
 
 source_urls = [GNU_SOURCE]
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['ab2633921a5cd38e48797bf5521ad259bdc4b979078034a3b790d7fec5493fab']
+
+patches = ['M4-1.4.18_glibc_2.28.patch']
+checksums = [
+    'ab2633921a5cd38e48797bf5521ad259bdc4b979078034a3b790d7fec5493fab',  # m4-1.4.18.tar.gz
+    'a613c18f00b1a3caa46ae4b8b849a0f4f71095ad860f4fcd6c6bb4ae211681fa',  # M4-1.4.18_glibc_2.28.patch
+]
 
 # '-fgnu89-inline' is required to avoid linking errors with older glibc's,
 # see https://github.com/easybuilders/easybuild-easyconfigs/issues/529
 configopts = "--enable-c++ CPPFLAGS=-fgnu89-inline"
 
+
```

Comment:
@robertschade Style nitpicking: no need to add a 2nd empty line here, please remove this again?

Reply:
Sorry, I had no time at the weekend. I included the change below.

Reply:
No need to apologize for not working on the weekend, please. ;)